### PR TITLE
ticdc: add missing links

### PR DESCRIPTION
**Pull request by overvenus**
### What is changed, added or deleted? (Required)

Add ticdc links back, try to keep the same as other TiDB tools.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: N/A
- Other reference link(s): N/A

### CODE REVIEW

File: TOC.md [hunk 1]
```diff
@@ -186,7 +186,11 @@
     + [故障诊断](/troubleshoot-tidb-lightning.md)
     + [FAQ](/tidb-lightning/tidb-lightning-faq.md)
     + [术语表](/tidb-lightning/tidb-lightning-glossary.md)
-  + [TiCDC](/ticdc/ticdc-overview.md)
+  + TiCDC
+    + [概述](/ticdc/ticdc-overview.md)
+    + [部署和任务管理](/ticdc/manage-ticdc.md)
```
Comment (TomShawn) [inline TOC.md:191]: Loos like this is duplicate with Line 71 in this TOC
@kissmydb PTAL

Comment (kissmydb) [reply TOC.md:191]: 之前段老板的建议不要在 TOC 增加重复的链接，我觉得也合理

Comment (TomShawn) [inline TOC.md:191]:
```suggestion
```

**Comment by TomShawn** [comment]
This change is not suitable.

**Pull request closed without merging**